eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -0,0 +1,23 @@
+steps:
+  - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+    - pwsh: |
+        $PrSHA = "$(Build.SourceVersion)"
+        $masterSHA = git rev-parse origin/master
```

Comment:
We should not assume master. We should use the source and target branch. See https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/get-markdown-files-from-changed-files.ps1#L5

I think you can do something as simple as:
```
git diff "origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" HEAD --name-only -- 'eng/common/*'
```